hibernate-core/src/main/java/org/hibernate/cfg/Settings.java

```diff
@@ -254,7 +254,8 @@ public boolean isAutoEvictCollectionCache() {
 
 	public boolean isAutoCreateSchema() {
 		return sessionFactoryOptions.getSchemaAutoTooling() == SchemaAutoTooling.CREATE
```

Comment:
Why not a method like `SchemaAutoTooling#isCreationEnabled()`?  That helps alleviate problems like this in the future.

Reply:
Well, all this is deprecated and useless AFAICS so I don't think it's worth doing more work on it.

Frankly, I was tempted to just remove all this but I suppose Settings is API and we can't do that.